evennia/accounts/accounts.py

```diff
@@ -1393,6 +1393,90 @@ def at_look(self, target=None, session=None, **kwargs):
             look_string = ("-" * 68) + "\n" + "".join(result) + "\n" + ("-" * 68)
             return look_string
 
+    def return_examine(self, viewer, *args, **kwargs):
+        """
+        Generates basic information about Accounts for @examine to see.
+        Args:
+            viewer: The Account or Character examining this object.
+            *args:
+            **kwags:
+        Returns:
+            results (str): The text to display.
+        """
+        message = list()
+        message.append(f"|wName/key|n: |c{self.name}|n ({self.dbref})")
+        aliases = self.aliases.all()
+        if aliases:
+            message.append("|wAliases|n: %s" % (", ".join(make_iter(str(self.aliases)))))
+        sessions = self.sessions.all()
+        if sessions:
+            message.append("|wSession id(s)|n: %s" % (", ".join("#%i" % sess.sessid
+                                                                for sess in self.sessions.all())))
+        message.append(f"|wEmail|n: |c{self.email}|n")
+        message.append(f"|wTypeclass|n: {self.typename} ({self.typeclass_path})")
+        perms = self.permissions.all()
+        perms_string = (", ".join(perms)) if perms else '<None>'
+        if self.is_superuser:
+            perms_string += " [Superuser]"
+        message.append(f"|wPermissions|n: {perms_string}")
+        message.append(self.locks.return_examine())
+        cmdsets = kwargs.pop('cmdsets', None)
+        if cmdsets:
+            message.append(self.return_examine_commands(viewer, cmdsets, *args, **kwargs))
+        message.append(self.scripts.return_examine())
+        message.append(self.tags.return_examine())
+        message.append(self.attributes.return_examine())
+        message.append(self.nattributes.return_examine())
+        return "\n".join(str(l) for l in message if l)
+
+
+    def return_examine_commands(self, viewer, cmdsets, *args, **kwargs):
```

Comment:
There is no reason to break this out into a separate hook like this. Either it should be a part of the `return_examine` hook, or this display method should go on the CmdSetHandler and be called from `return_examine` maybe.